Labyrinth.Console/Program.cs

```diff
@@ -38,28 +41,59 @@
     {
         ClearPlayer();
         playerCoordinates = playerCoordinates with { Y = playerCoordinates.Y - 1 };
-        RenderPlayer();
+        if(!BannedCoordinates.Contains(playerCoordinates))
```

Comment:
This line is duplicated four times. Can you think of a way to extract the common logic within a method maybe?